### ISSUE TITLE
[Question]: About Splatfests

### ISSUE THREAD

**Issue by ollietheotter983** [PretendoNetwork/splatoon#8]
### Checked Existing

- [X] I have checked the repository for duplicate issues.

### What would you like to know?

When Splatfests are fully working, will they be hosted by Pretendo? That would be great (especially if working using SSSL) Perhaps now SSSL is around they should test one again?

**Comment by ssdrive-flaktra** [comment]
SSSL is sort of broken with Splatoon right now. Splatfests are handled in BOSS, not this server. But, you're gonna want to use the branch `feat/fests` to fix a bug w/ fests.

**Comment by ollietheotter983** [comment]
Is that branch accessible via SSSL?

**Comment by ssdrive-flaktra** [comment]
> Is that branch accessible via SSSL?

You misunderstand. The branch is like a specific version of the splatoon server. Currently, splatoon does not work with SSSL. You need a modded Wii U.

**Comment by ollietheotter983** [comment]
I've been playing splatoon with SSSL for months, that is if SSSL is the thing where you change your dns server. My console is completely vanilla without mods and has been since launch

**Comment by ssdrive-flaktra** [comment]
> I've been playing splatoon with SSSL for months, that is if SSSL is the thing where you change your dns server. My console is completely vanilla without mods and has been since launch

Well if you want splatfests you're going to need to self host:
- account server
- friends server
- splatoon server

you need a linux server for all of those

**Comment by ollietheotter983** [comment]
The question was really does Pretendo plan on ever hosting them sorry if this sounds rude

**Comment by MatthewL246** [comment]
Yes, bug fixes in other branches will be deployed to the live servers eventually, but we don't have an ETA on that.

**Comment by ssdrive-flaktra** [comment]
> Yes, bug fixes in other branches will be deployed to the live servers eventually, but we don't have an ETA on that.

yeah, especially with the new matchmaking rewrite

**Comment by ollietheotter983** [comment]
> yeah, especially with the new matchmaking rewrite

just purely out of curiousity, why are they doing a matchmaking rewrite?

**Comment by Pinuaple** [comment]
bcs i think current one is super de equilibrated
